yaksh/python_assertion_evaluator.py

```diff
@@ -49,6 +49,8 @@ def check_code(self, user_answer, file_paths, test_case):
             err = "{0} {1} in: {2}".format(type.__name__, str(value), text)
         except TimeoutException:
             raise
+        except Exception:
```

Comment:
In theory couldn't you remove both this and the previous except block as the caller will anyway catch and report the exception?